NGitLab/Models/ReleaseLinkCreate.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Runtime.Serialization;
+
+namespace NGitLab.Models
+{
+    [DataContract]
+    public class ReleaseLinkCreate
+    {
+        [DataMember(Name = "name")]
+        public string Name;
```

Comment:
The project is pretty old and the original code didn't use properties. That's why most classes have public fields. But we do prefer using properties for new classes. You could use auto properties such as:

```suggestion
        public string Name { get; set; }
```

Reply:
Applied to all the new classes